src/Inventory.zig

```diff
@@ -425,7 +427,7 @@ pub const Sync = struct { // MARK: Sync
 		}
 	};
 
-	pub fn addHealth(health: f32, cause: main.game.DamageType, side: Side, id: u32) void {
+	pub fn addHealth(health: f32, cause: main.game.DamageType, side: Side, id: EntityIndex) void {
```

Comment:
Introduction of the EntityIndex type can be done in a separate PR, to reduce the amount of lines to review here.